bindings/wallet-cordova/src/ios/WalletPlugin.m

```diff
@@ -264,6 +262,12 @@ - (void)WALLET_VOTE:(CDVInvokedUrlCommand*)command
     NSString* settings_ptr_raw = [command.arguments objectAtIndex:1];
     NSString* proposal_ptr_raw = [command.arguments objectAtIndex:2];
     NSString* choice_raw = [command.arguments objectAtIndex:3];
+    NSDictionary* expirationDate = [command.arguments objectAtIndex:4];
```

Comment:
these changes don't actually belong here, but it was the faster to just fix them here to see if the ios pipeline did work.